scripts/run_integration_tests.sh

```diff
@@ -69,7 +69,7 @@ source $(dirname $0)/setup.sh || exit 1
 source $(dirname $0)/setup_gae.sh || exit 1
 
 # Install third party dependencies
-bash scripts/install_third_party.sh
+bash -x scripts/install_third_party.sh
```

Comment:
Travis aborts test runs if there is no output to STDOUT for a certain amount of time. IIRC, this is a workaround to prevent that. It prints the current line being executed to the STDOUT to let Travis know that our tests are not hanging.